resources/views/movies/movieinfo.blade.php

```diff
@@ -0,0 +1,205 @@
+                            <section class="col-auto">
+                                <p>Movie</p>
+                            </section>
+                            <section class="col-auto">
+                                <p>{{$movie->release_date}}</p>
+                            </section>
+                            <section class="col-auto">
+                                <p>02h 43m</p>
+                            </section>
+                        </article>
+                    </article>
+                </section>
+
+                <section class="col-auto offset-5">
+                    <article class="row">
+                        <h6>SMDB RATING</h6>
+                    </article>
+                    <article class="row">
+                        <section class="col-auto">
+                            <img src="https://uxwing.com/wp-content/themes/uxwing/download/arts-graphic-shapes/star-symbol-icon.png"
+                            style="width: 45px;">
+                        </section>
+                        <section class="col">
+                            <article class="row">
+                                <p class="ratingP">{{number_format($movie->ratings->avg('movie_rating.rating'), 1)}}/10</p>
+                            </article>
+                            <article class="row">
+                                <p class="ratingP">{{$movie->ratings->count() }}</p>
+                            </article>
+                        </section>
+                    </article>
+                </section>
+                <section class="col">
+                    <article class="row">
+                            <h6>YOUR RATING</h6>
+                    </article>
+                    <article class="row">
+                        <section class="col-auto">
+                            <button class="rateButton" data-bs-toggle="modal" data-bs-target="#ratingModal">
+                                <article class="row">
+                                    <section class="col-auto">
+                                        <img src="https://cdn-icons-png.flaticon.com/512/9784/9784192.png"
+                                        style="width: 45px;">
+                                    </section>
+                                    <section class="col-auto ratingP">
+                                        <h5 class="rate">Rate</h5>
+                                    </section>
+                                </article>
+                            </button>
+                        </section>
+                    </article>
+                </section>
+            </article>
+        </div>
+        
+        <div class="container-fluid">
+            <article class="row">
+                <section class="col-3 offset-1">
+                    @isset($movie->poster_url)
+                        <img src="{{$movie->poster_url}}" class="pic">  
+                    @endisset
+                </section>
```

Comment:
Maybe wrap isset around the section tag, so no empty section tag is generated if no poster url.